src/components/gitOps/GitOpsConfiguration.tsx

```diff
@@ -31,8 +34,32 @@ const DefaultGitOpsConfig = {
     username: "",
     gitLabGroupId: "",
     gitHubOrgId: "",
+    azureOrgId: "",
     active: true,
 }
+
+const GitProviderTabIcons: React.FC<{ gitops: string }> = ({ gitops }) => {
+    switch (gitops) {
+        case "GitHub": return <GitHub />
+        case "GitLab": return <GitLab />
+        case "Azure": return <img className="icon-dim-24" src={Azure} />
```

Comment:
Why different handling compared to above

Reply:
fixed `Àzure` by using SVG insted png